src/test/client/api.vscode.test.ts

```diff
@@ -52,14 +52,14 @@ suite('3rd Party Kernel Service API', function () {
             await startJupyterServer();
             traceInfo(`Start Test (completed) ${this.currentTest?.title}`);
         } catch (e) {
-            await captureScreenShot(this.currentTest?.title || 'unknown');
+            await captureScreenShot(this);
```

Comment:
This was repeated everywhere, now its simpler we just call `captureScreenShot`, 
and has the added benefit of being able to predict the file names for test failures, so we can attach them to the ipynb file.